sdk/python/kfp/components/executor_test.py

```diff
@@ -149,10 +153,15 @@ def test_func(output_parameter_path: OutputPath(str)):
                 output_parameter_path,
                 'gs://some-bucket/some_task/nested/output_parameter')
```

Comment:
Actually, this expected value is wrong here. `OutputPath` expects a local file path, while `OutputUri` could be a URI like `gs://....`.  Can you fix the expected data to be a local path, something like `/gcs/some-bucket/...`?